### ISSUE TITLE
Azure - Grafana+Prometheus deployments fail with "Addition of a managed disk to a VM with blob based disks is not supported."

### ISSUE THREAD

**Issue by shaharyakir** [camilb/prometheus-kubernetes#51]
I'm running on Azure with acs-engine.

The deployment script ends successfully, but the pods are not able to start:

`Warning  FailedMount            15s (x2 over 1m)  attachdetach                          AttachVolume.Attach failed for volume "pvc-f1da4bbd-f6dc-11e7-a367-000d3a1b19fd" : Attach volume "XXX-dynamic-pvc-f1da4bbd-f6dc-11e7-a367-000d3a1b19fd" to instance "k8s-nodepool1-18946821-0" failed with compute.VirtualMachinesClient#CreateOrUpdate: Failure responding to request: StatusCode=409 -- Original Error: autorest/azure: Service returned an error. Status=409 Code="OperationNotAllowed" Message="Addition of a managed disk to a VM with blob based disks is not supported."`

**Comment by camilb** [comment]
@shaharyakir I'm not very familiar with Azure, but willing to fix this.  `acs-engine` and  `Azure Container Service (AKS)` are the same now? I remember Azure supporting Kubernetes for quite long time, but only tested AKS recently and it was on preview.

**Comment by shaharyakir** [comment]
no, AKS is the very-limited, Microsoft official implementation.

acs-engine is open source and configurable.

**Comment by shaharyakir** [comment]
Ok, will check. Thank you.

**Comment by shaharyakir** [comment]
@sjdweb Works. Thanks!
